packages/tools/kadena-cli/src/account/accountOptions.ts

```diff
@@ -213,9 +214,9 @@ export const accountOptions = {
     defaultIsOptional: false,
     prompt: account.publicKeysForAccountAddPrompt,
     expand: async (publicKeys: string): Promise<string[]> => {
-      const keys = publicKeys.split(',');
+      const keys = publicKeys?.split(',');
       return keys
-        .map((key: string) => key.trim())
+        ?.map((key: string) => key.trim())
         .filter((key: string) => !isEmpty(key));
```

Comment:
how would a empty key be still available here? shouldnt it be restricted at validation?

Reply:
when you pass --quiet flag and it's an optional field. it could be empty.